packages/shared/src/referralCode/type.ts

```
@@ -0,0 +1,101 @@
+      symbol: string;
+    };
+    amount: string;
+    fiatValue: string;
+  };
+  items: IHardwareSalesRecordItem[];
+}
+
+export interface IInviteHistory {
+  total: number;
+  items: IHardwareSalesRecordItem[];
+}
```

ESLint

[error] Insert `⏎`

(prettier/prettier)